src/main/java/DialogBox.java

```diff
@@ -0,0 +1,127 @@
+//import javafx.collections.FXCollections;

```

Comment:
Please remove the commented code blocks. You can always refer to your previous version of the code in previous commits. Therefore there is no need to retain commented code segments.